### DISCUSSION TITLE
deploy token error

### DISCUSSION THREAD

**Discussion by mr-ask-chay** [peaceiris/actions-gh-pages#1003] [Q&A]
My app got deployed to gh-pages successfully by the yml but the page is white, By doing with the deploy into gh-pages with manually the site is working fine, doing it with yml is not working, and I will provide you the yml link here so that you can see my configuration https://github.com/otaku0304/password-remover-for-pdf/blob/master/.github/workflows/gh-pages.yml

**Comment by peaceiris** [comment]
Has your problem been resolved now?

**Reply by mr-ask-chay** [reply]
no, after successful deployment the site is not working but doing it manually is working. through yml page is white.
